pkg/neorpc/result/invoke.go

```diff
@@ -99,25 +105,25 @@ func (r *Iterator) UnmarshalJSON(data []byte) error {
 	if err != nil {
 		return err
 	}
+	if iteratorAux.Value != nil {
+		r.Values = make([]stackitem.Item, len(iteratorAux.Value))
+		for j := range r.Values {
+			r.Values[j], err = stackitem.FromJSONWithTypes(iteratorAux.Value[j])
+			if err != nil {
+				return fmt.Errorf("failed to unmarshal iterator values: %w", err)
+			}
+		}
+	}
+	r.Truncated = iteratorAux.Truncated
```

Comment:
And then if you move this code below and place it exactly after `if len(iteratorAux.Interface) != 0` block, then the inner code block (`r.Values = make([]stackitem.Item,...`) won't be marked as "changed" by `git diff`, and that's exactly what I'm looking for when I ask not to perform useless changes.

The only thing that will be affected is, in fact, the `} else {` line at L112 and the `r.Truncated = iteratorAux.Truncated` condition that should be moved out of this block.